.devcontainer/.env

```diff
@@ -0,0 +1,5 @@
+PLAYWRIGHT_IMAGE=mcr.microsoft.com/playwright
+PLAYWRIGHT_VERSION=v1.46.0
```

Comment:
We can set this value to `v1.49.1` as that is the current version being used. I know the PLAYWRIGHT_VERSION can be overwritten but let's synchronize the version used in the project with the default value in this file.

Reply:
Thanks for review @carlosthe19916. I moved `.env` and `compose.yaml` under the `/etc/playwright-compose` directory as you suggested. Regarding the Markdown formatting, I followed [this guide](https://docs.gruntwork.io/guides/style/markdown-style-guide/), I don't use IDE when writing Markdown/adoc/rst documents, just simple text editor like Geany (I used Atom before, but it was deprecated).

Version of Playwright updated to `v1.49.1` as suggested.